apps/mobile-wallet/src/components/NFTThumbnail.tsx

```diff
@@ -92,6 +134,37 @@ const NFTThumbnail = ({ nft, size }: NFTThumbnailProps) => {
   )
 }
 
+const NoImagePlaceholder = ({ size }: Pick<NFTThumbnailProps, 'size'>) => (
+  <NoImage style={{ width: size, height: size }}>
+    <CameraOff color="gray" />
+  </NoImage>
+)
+
+const WebViewImage = ({ nft, size }: NFTThumbnailProps) => (
+  <WebView
+    source={{ html: `<img src="${nft.image}" />` }}
+    style={{ width: size, height: size, borderRadius: BORDER_RADIUS_SMALL, backgroundColor: 'transparent' }}
+    injectedJavaScript={
+      "const meta = document.createElement('meta'); meta.setAttribute('content', 'width=width, initial-scale=1, maximum-scale=1, user-scalable=1'); meta.setAttribute('name', 'viewport'); document.getElementsByTagName('head')[0].appendChild(meta); "
+    }
+    javaScriptEnabled={false}
```

Comment:
So this should avoid any injection attack, correct?
And can you confirm that all that can be rendered in this webview is an <img/> tag?

Reply:
The only user content that we are rendering is the `nft.image`, which I am sanitizing. So, I am fairly confident that we are safe. I don't see how anything could be injected in the webview